gammapy/catalog/fermi.py

```diff
@@ -56,22 +56,226 @@ class SourceCatalogObject3FGL(SourceCatalogObject):
+        elif tevcatFlag == 'P':
+            tevcatMessage = 'Small TeV source'
+        elif tevcatFlag == 'E':
+            tevcatMessage = 'Extended TeV source (diameter > 40 arcmins)'
+        else:
+            tevcatMessage = 'N/A'
+        ss += '{:<30s} : {}\n'.format('TeVCat flag', tevcatMessage)
+
+        flagMessage = {
+            0: 'None',
+            1: 'Source with TS > 35 which went to TS < 25 when changing the diffuse model. Note that sources with TS < '
+               '35 are not flagged with this bit because normal statistical fluctuations can push them to TS < 25.',
+            3: 'Flux (> 1 GeV) or energy flux (> 100 MeV) changed by more than 3 sigma when changing the diffuse model.'
+               ' Requires also that the flux change by more than 35% (to not flag strong sources).',
+            4: 'Source-to-background ratio less than 10% in highest band in which TS > 25. Background is integrated '
+               'over the 68%-confidence area (pi*r_682) or 1 square degree, whichever is smaller.',
+            5: 'Closer than theta_ref from a brighter neighbor, where theta_ref is defined in the highest band in which'
+               ' source TS > 25, or the band with highest TS if all are < 25. theta_ref is set to 2.17 degrees (FWHM)'
+               ' below 300 MeV, 1.38 degrees between 300 MeV and 1 GeV, 0.87 degrees between 1 GeV and 3 GeV, 0.67'
+               ' degrees between 3 and 10 GeV and 0.45 degrees about 10 GeV (2*r_68).',
+            6: 'On top of an interstellar gas clump or small-scale defect in the model of diffuse emission. This flag '
+               'is equivalent to the "c" suffix in the source name.',
+            7: 'Unstable position determination; result from gtfindsrc outside the 95% ellipse from pointlike.',
+            9: 'Localization Quality > 8 in pointlike (see Section 3.1 in catalog paper) or long axis of 95% ellipse >'
+               ' 0.25.',
+            10: 'Spectral Fit Quality > 16.3 (see Equation 3 in 2FGL catalog paper).',
+            11: 'Possibly due to the Sun (see Section 3.6 in catalog paper).',
+            12: 'Highly curved spectrum; LogParabola beta fixed to 1 or PLExpCutoff Spectral Index fixed to 0 (see '
+                'Section 3.3 in catalog paper).'
+        }
+        ss += '{:<30s} : {}\n'.format('Other flags', flagMessage.get(d['Flags'], 'N/A'))
+
+        return ss
+
+    def _info_position(self):
+        """Print position info."""
+        d = self.data
+        ss = '\n*** Position info ***\n\n'
+        ss += '{:<20s} : {:.3f} deg\n'.format('RA (J2000)', d['RAJ2000'])
+        ss += '{:<20s} : {:.3f} deg\n'.format('Dec (J2000)', d['DEJ2000'])
+        ss += '{:<20s} : {:.3f} deg\n'.format('GLON', d['GLON'])
+        ss += '{:<20s} : {:.3f} deg\n'.format('GLAT', d['GLAT'])
 
-        ss = 'Source: {}\n'.format(d['Source_Name'])
         ss += '\n'
+        ss += '{:<20s} : {:.0f}\n'.format('ROI number', d['ROI_num'])
```

Comment:
Suggest to put ROI number last in this section.
It's not something a user usually cares about.